src/SFA.DAS.EAS.Web/SFA.DAS.EAS.Web.csproj

```diff
@@ -114,7 +114,6 @@
     <Compile Include="ViewModels\PaymentTransactionViewModel.cs" />
     <Compile Include="ViewModels\PrivacyViewModel.cs" />
     <Compile Include="ViewModels\EmployerAccountPayeListViewModel.cs" />
-    <Compile Include="ViewModels\EmployerAgreementViewModel.cs" />

```

Comment:
Technically Removing Agreement related models and files aren't part of the optimisations. I know EAS.Web is overdue some thinning but in future i'd prefer to keep refactoring fairly focused

Reply:
The tests have been moved, along with the logic, to the command handlers.